ambry-router/src/main/java/com/github/ambry/router/NonBlockingRouter.java

```diff
@@ -131,6 +131,14 @@ class NonBlockingRouter implements Router {
     routerMetrics.initializeQuotaOCMetrics(ocList);
   }
 
+  Integer incrementAndGetOperationCount() {
+    return currentOperationsCount.incrementAndGet();
```

Comment:
Is it worth writing a function just for one line?  Sometimes it's easier to see the actual line.  One example case of wrapping a one liner in a function is when the name makes the code easier to understand due to bad variable name.  Another case is when unit test needs to mock that method to make it easier to test.  Another case is it's intended to be overwritten.

Reply:
makes sense